translate/storage/tbx.py

```diff
@@ -113,6 +139,21 @@ def _getnotelist(self, origin=None):
     def getnotes(self, origin=None):
         return "\n".join(self._getnotelist(origin=origin))
 
+    def istranslatable(self) -> bool:
+        for node in self._getnotenodes(origin="definition"):
+            if self._is_translation_needed_node(node):
+                return self._getnodetext(node).strip().lower() == "yes"
+        return super().istranslatable()
+
+    def isobsolete(self) -> bool:
+        for note in self._getnotenodes(origin="pos"):
+            if self._is_administrative_status_term_node(note) and self._getnodetext(
+                note
+            ).strip().lower() in {"forbidden", "obsolete"}:
+                return True
+
+        return super().isobsolete()
```

Comment:
I'm not sure if this is a good mapping. `isobsolete` units are currently not imported into Weblate at all because these contain strings that were translated in the past and are no longer used. They are stored in the translation files so that they can be reused in case the string re-appears (kind of translation memory). See also https://github.com/WeblateOrg/weblate/issues/1993.

Reply:
Yes, that sounds as a good approach.